terraform/modules/neptune/main.tf

```diff
@@ -0,0 +1,77 @@
+resource "aws_neptune_cluster" "neptune_cluster" {
+  apply_immediately                    = true
+  backup_retention_period              = var.backup_retention_period
+  cluster_identifier                   = "${var.resource_prefix}-neptune-cluster"
+  deletion_protection                  = var.deletion_protection
+  enable_cloudwatch_logs_exports       = ["audit"]
+  engine                               = "neptune"
+  engine_version                       = var.neptune_engine_version
+  iam_roles                            = var.neptune_iam_roles
+  neptune_subnet_group_name            = var.create_neptune_subnet_group ? aws_neptune_subnet_group.this[0].name : var.db_subnet_group_name
+  port                                 = 8182
```

Comment:
since the value for local.neptune_port is defined we should use this for the "port" setting in the neptuine cluster, currently this is hardcoded. also, the log setting should be controlled by a variable with a default setting - even though this only currently supports "audit" this may change in the future and we don't want to limit this configuration